source/administration-guide/upgrade/important-upgrade-notes.rst

```diff
@@ -6,9 +6,91 @@ Important Upgrade Notes
+|                                                    | plugin enabled can remove it as it no longer sends the feedback over through telemetry.                                                                          |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Format query parameter requirement in ``/api/v4/config/client`` endpoint has been deprecated.                                                                    |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Deprecated mmctl commands and flags have been removed.                                                                                                           |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Experimental certificate-based authentication feature has been removed. ``ExperimentalSettings.ClientSideCertEnable`` must be ``false`` to start the server.     |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Added logic to migrate the password hashing method from bcrypt to PBKDF2. The migration will happen progressively, migrating the password of a user as soon      |
+|                                                    | as they enter it; e.g. when logging in or when double-checking their password for any sensitive action. There is an edge case where users might                  |
+|                                                    | get locked out of their account: if a server upgrades to v11 and user A logs in (i.e., they need to enter their password), and then the server downgrades        |
+|                                                    | to v10.12 or previous, user A will no longer be able to log in. In this case, admins will need to manually reset the password of such users, through the         |
+|                                                    | system console or through the ``mmctl user reset-password [users]`` command.                                                                                     |
```

Comment:
Maybe we can link to https://docs.mattermost.com/administration-guide/manage/mmctl-command-line-tool.html#mmctl-user-reset-password here.